packages/components/src/components/hds/form/super-select/after-options.hbs

```diff
@@ -0,0 +1,27 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+{{#if @showNoSelectedMessage}}
```

Comment:
From @alex-ju comment

> I want to note that we have not accounted for all possible combinations because we are programmatically controlling most of the arguments for this subcomponent and don't expect this component to be used in isolation (it's only for us to avoid code repetition and share it between single and multi variants).

Resolving this comment then.